server/mediaserver.go

```diff
@@ -135,17 +129,13 @@ type authWebhookResponse struct {
 	VerificationFreq uint `json:"verificationFreq"`
 }
 
-func NewLivepeerServer(rtmpAddr string, lpNode *core.LivepeerNode, httpIngest bool, transcodingOptions string) (*LivepeerServer, error) {
+func NewLivepeerServer(lpNode *core.LivepeerNode, httpIngest bool, transcodingOptions string) (*LivepeerServer, error) {
 	opts := lpmscore.LPMSOpts{
-		RtmpAddr:     rtmpAddr,
-		RtmpDisabled: true,
```

Comment:
It either shouldn't be removed here, or RTMP support should be remove also in [LPMS](https://github.com/livepeer/lpms/blob/6e76b1c43b694088aaa57cd96b9bbf6d5abcdc1a/core/lpms.go#L75). Otherwise, O and T will listen for RTMP connections on startup.